src/main/java/com/github/pjfanning/xlsx/impl/ooxml/OoXmlStrictConverter.java

```diff
@@ -0,0 +1,227 @@
+package com.github.pjfanning.xlsx.impl.ooxml;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Properties;
+import javax.xml.namespace.QName;
+import javax.xml.stream.XMLEventFactory;
+import javax.xml.stream.XMLEventReader;
+import javax.xml.stream.XMLEventWriter;
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLOutputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.events.Attribute;
+import javax.xml.stream.events.EndElement;
+import javax.xml.stream.events.Namespace;
+import javax.xml.stream.events.StartElement;
+import javax.xml.stream.events.XMLEvent;
+import org.apache.poi.hssf.usermodel.HSSFAnchor;
+import org.apache.poi.hssf.usermodel.HSSFDataBarFormatting;
+import org.apache.poi.hssf.usermodel.HSSFDateUtil;
+
+public class OoXmlStrictConverter implements AutoCloseable {
+
+    private static final XMLEventFactory XEF = XMLEventFactory.newInstance();
+    private static final XMLInputFactory XIF = XMLInputFactory.newInstance();
+    private static final XMLOutputFactory XOF = XMLOutputFactory.newInstance();
+    private static final QName CONFORMANCE = new QName("conformance");
+    private static final Properties mappings;
+
+    static {
+        XOF.setProperty(XMLOutputFactory.IS_REPAIRING_NAMESPACES, true);
```

Comment:
this line won't be needed if you use the XMLHelper method to create an output factory

Reply:
Done